src/main/java/user/reg/abr/UserRegHelper.java

```diff
@@ -0,0 +1,44 @@
+package user.reg.abr;
+
+import user.database.UserRegisterDataBaseGateway;
+
+public class UserRegHelper {
+
+
+    public UserRegResponseModel register(UserRegRequestModel requestModel, UserRegResponseModel responseModel, UserRegisterDataBaseGateway dataBaseGateway){
+        String password = requestModel.getPassword();
+        String userName = requestModel.getUserName();
+        String rePassword = requestModel.getRePassword();
+
+        boolean passwordValidity = (checkCharacterValidity(password) & password.equals(rePassword));
```

Comment:
I suggest using && instead of & in the case that .equals() throws an excpetion (e.g. null exception).